keanu-project/src/main/java/io/improbable/keanu/tensor/dbl/DoubleTensor.java

```diff
@@ -245,6 +247,10 @@ static DoubleTensor placeHolder(int[] shape) {
 
     DoubleTensor standardizeInPlace();
 
+    DoubleTensor zero();
```

Comment:
I think we should implement both `zero` and `zeroInPlace` so that we're calling out when the tensor will be modified.

Reply:
Also note that we already have a static `zeros` method

Reply:
The static `zero` doesn't mutate - it creates.... I don't see why we'd create a non-static non-mutating zero?

Reply:
Agreed. To refine my request: can we call this method "zeroInPlace" so that users understand that it's mutating?